components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -820,12 +832,12 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
             hno3_cond(1,1,2), radius_strat(1,1,2), ncol, lchnk, 2 )
 #endif
 
-       !-----------------------------------------------------------------------      
+       !-----------------------------------------------------------------------
        !	... reform total hno3 = gas + all condensed
-       !-----------------------------------------------------------------------      
+       !-----------------------------------------------------------------------
        do k = 1,pver
           vmr(:,k,hno3_ndx) = vmr(:,k,hno3_ndx) + hno3_cond(:,k,1) &
-               + hno3_cond(:,k,2) 
+               + hno3_cond(:,k,2)
```

Comment:
There are some 'end do' and 'end if' instances here and to the end of this subroutine, but could be taken care of during a final refactor of mo_gas_phase_chemdr

Reply:
Thanks, @brigaudet. I am thinking of reverting changes to mo_gas_phase_chemdr.F90 and adding only the changes needed for this PR to avoid resolving conflicts again and again. We can address these in the final refactor of this module as you said.